ext/pdo/pdo_sqlstate.c

```diff
@@ -226,7 +226,7 @@ static const struct pdo_sqlstate_info err_initializer[] = {
 	{ "54023",	"Too many arguments" },
 	{ "55000",	"Object not in prerequisite state" },
 	{ "55006",	"Object in use" },
-	{ "55P02",	"Cant change runtime param" },
+	{ "55P02",	"Can't change runtime param" },
```

Comment:
BC change for those who expect this message maybe? :thinking:

Reply:
@carusogabriel shouldn't really matter tbh, if you are expecting an error message you are doing something wrong anyway tbh (especially for tests outside php-src)

Reply:
Hm, yes... This is a part 2/3 of typos fixing with `codespell` script... I can remove this particular change for the error message if that's the case of BC break...